dev/sg/internal/run/run.go

```diff
@@ -280,6 +289,7 @@ func runWatch(
 			cmdOut, err := BashInRoot(ctx, cmd.Install, makeEnv(globalEnv, cmd.Env))
 			if err != nil {
 				if !startedOnce {
+					installDone <- cmd.Name
```

Comment:
Minor suggestion since the variable was also renamed at the call site of `runWatch`:

```suggestion
					installed <- cmd.Name
```

Note, if you choose to accept this, the argument in the function definition will also need to be changed on line 257, but GitHub doesn't let me add that as a suggestion.